src/lib/pickles_types/nat.mli

```diff
@@ -1,4 +1,4 @@
-(** Representation of naturals for Pickles *)
+(** Representation of naturals for Pickles with Peano encoding *)
```

Comment:
This encoding could be inefficient, why is it used?

Reply:
Opened https://github.com/MinaProtocol/mina/issues/14193